test/unit/startup-test.js

```diff
@@ -165,19 +169,21 @@ describe('Startup tests', function() {
             iotagentJSON.start(iotAgentConfig, function(error) {
                 should.not.exist(error);
 
-                mqtt.connect.calledOnceWithExactly({
-                    ca: 'cacontent',
-                    cert: 'certcontent',
-                    connectTimeout: 3600000,
-                    host: '127.0.0.1',
-                    keepalive: 0,
-                    key: 'keycontent',
-                    password: null,
-                    port: '8883',
-                    protocol: 'mqtts',
-                    rejectUnauthorized: true,
-                    username: null
-                }).should.equal(true);
+                mqtt.connect
```

Comment:
Not sure if I like this... too "sparse", don't you think?

On the contrary

```
mqtt.connect.calledOnceWithExactly({
```

looks nice.

Reply:
Fixed 7657825